### PR TITLE
Make onItemRemoved provider notification work

### PR DESCRIPTION
**Pull request by NeilRashbrook**

### CODE REVIEW

File: calendar/experiments/calendar/parent/ext-calendar-provider.js [hunk 1]
```diff
@@ -355,7 +355,10 @@ this.calendar_provider = class extends ExtensionAPI {
             name: "calendar.provider.onItemRemoved",
             register: fire => {
               let listener = (event, calendar, item) => {
-                return fire.async(convertCalendar(context.extension, calendar), item);
+                return fire.async(
+                  convertCalendar(context.extension, calendar),
+                  convertItem(item, {}, context.extension),
```
Comment (kewisch) [inline calendar/experiments/calendar/parent/ext-calendar-provider.js:360]: If you don't mind, could you add `extraParameters` like for `onItemUpdated` and pass that in as options?

Comment (NeilRashbrook) [reply calendar/experiments/calendar/parent/ext-calendar-provider.js]: Done.

**Comment by benbucksch** [comment]
done

**Comment by benbucksch** [comment]
Review comment addressed

**Comment by kewisch** [comment]
I didn't look at the pull requests before I pushed, seems I re-noticed this happening. Addressed the last remaining piece, so we are covered here.

**Pull request closed without merging**